x/ccv/types/events.go

```diff
@@ -2,11 +2,38 @@ package types
 
 // CCV events
 const (
-	EventTypeTimeout      = "timeout"
-	EventTypePacket       = "ccv_packet"
-	EventTypeChannelClose = "channel_closed"
+	EventTypeTimeout                  = "timeout"
+	EventTypePacket                   = "ccv_packet"
+	EventTypeChannelClose             = "channel_closed"
+	EventTypeChannelEstablished       = "channel_established"
+	EventTypeFeeTransferChannelOpened = "fee_channel_opened"
+	EventTypeConsumerClientCreated    = "consumer_client_created"
+
+	EventExecuteConsumerChainSlash = "execute_consumer_chain_slash"
```

Comment:
Should line 12 have `EventType` as a prefix?

Reply:
Added.